adapter/internal/operator/apis/dp/v1alpha1/api_webhook.go

```diff
@@ -163,12 +152,51 @@ func (r *API) validateFormats() *field.Error {
 	if errMsg := validateContext(r.Spec.Context); errMsg != "" {
 		return field.Invalid(field.NewPath("spec").Child("context"), r.Spec.Context, errMsg)
 	}
+	if errMsg := validateAPIDisplayName(r.Spec.APIDisplayName); errMsg != "" {
```

Comment:
Shall we return all the errors at the end, so the user can see and correct all the errors in the API resource at once?

Reply:
It feels cleaner not to combine all errors together and return a simple message so a user can identify the error and the section clearly.

Reply:
I agree it's better to send it at once. I'll fix this